archive-tar.c

```diff
@@ -38,11 +37,18 @@ static int write_tar_filter_archive(const struct archiver *ar,
 #define USTAR_MAX_MTIME 077777777777ULL
 #endif
 
+/* writes out the whole block */
+static void write_block(const char * block) {
+	if (gzip) {
+		if (gzwrite(gzip, block, BLOCKSIZE) != BLOCKSIZE) error("gzwrite failed");
```

Comment:
In Git's source code, we never have conditional statements on the same line as the `if` keyword. Please always put the conditional statements into their own line, whether with or without curlies is up to you, but it needs to be an own line.